web/vtadmin/src/components/routes/workflow/WorkflowStreams.tsx

```diff
@@ -122,29 +122,6 @@ export const WorkflowStreams = ({ clusterID, keyspace, name }: Props) => {
                     <WorkflowStreamsLagChart clusterID={clusterID} keyspace={keyspace} workflowName={name} />
                 </>
             )}
-
```

Comment:
Can you take a look at the lint errors: looks like we can remove some imports and variables/constants after deleting the lines below.